src/ts/components/button.ts

```diff
@@ -16,6 +16,11 @@ export interface ButtonConfig extends ComponentConfig {
    * WCAG20 standard for defining info about the component (usually the name)
    */
   ariaLabel?: string;
+
+  /**
+   * WCAG20 standard to display if a button is pressed or not
+   */
+  ariaPressed?: string;
```

Comment:
Can we change this to `boolean` and make the `toString` internally? Or why is this a `string` if it's suffixed with `pressed`